oneandone/src/main/java/org/apache/jclouds/oneandone/rest/config/OneAndOneProperties.java

```diff
@@ -19,8 +19,9 @@
 public final class OneAndOneProperties {
 
    public static final String POLL_PREDICATE_SERVER = "jclouds.oneandone.rest.predicate.server";
+   public static final String POLL_PREDICATE_SERVER_RUNNING = "jclouds.oneandone.rest.predicate.serveron";
    public static final String POLL_PREDICATE_PRIVATE_NETWORK = "jclouds.oneandone.rest.predicate.privatenetwork";
-   public static final String AUTH_TOKEN = "oneandone.identity";
+   public static final String AUTH_TOKEN = "oneandone.crednetial";
```

Comment:
Remove this property and the line that uses it in the live test base class. jclouds already sets it to the right value [here](https://github.com/jclouds/jclouds/blob/master/core/src/test/java/org/jclouds/apis/BaseApiLiveTest.java#L60).